src/celengine/observer.cpp

```diff
@@ -578,14 +578,14 @@ Observer::setOriginalOrientation(const Eigen::Quaterniond& q)
     updateOrientation();
 }
 
-Eigen::Matrix3d
+Eigen::Quaterniond
```

Comment:
Maybe it would be useful to return this as `const Eigen::Quaterniond&` to avoid forcing the copy?